src/main_operations/definitions/mx3_async_object_detection.py

```
@@ -0,0 +1,200 @@
+        """Close this operation's accelerator stream."""
+        if self.binding is not None:
+            self.binding.close()
+
+    def unbind(self) -> None:
+        """Return this operation's reserved stream to the shared coordinator."""
+        if self.binding is None:
+            return
+        binding, self.binding = self.binding, None
+        try:
+            self.coordinator.unregister_stream(binding)
+        except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)